integration/quarkus/deployment/src/main/java/com/blazebit/persistence/integration/quarkus/deployment/BlazePersistenceProcessor.java

```diff
@@ -148,6 +166,15 @@ void reflection(EntityViewsBuildItem entityViewsBuildItem,
                 );
             }
         }
+
+        // add fromString methods from entity view id types
+        for (EntityViewMapping entityViewMapping : entityViewConfigurationBuildItem.getEntityViewConfiguration().getEntityViewMappings()) {
+            reflectionProducer.produce(ReflectiveClassBuildItem.builder(
+                    entityViewMapping.getIdAttribute().getDeclaredType())
+                    .methods(true)
```

Comment:
Can't you provide a specific method signature instead?

Reply:
No, the API does not allow for this.